frontend/www/src/js/modules/dashboard/tradition/propertyTableView.js

```diff
@@ -3,13 +3,14 @@
  *   owner: string;
  *   access: string;
  *   language: string;
+ *   direction: string;
  *   tradition: string;
  *   witnesses: string;
  * }} TraditionMetaLabels
  *
  * @typedef {{ stemma: string }} StemmaMetaLabels
  *
- * @typedef {{ label: string; value: string }} MetaItem
+ * @typedef {{ label: string; value: string, inputOptions: InputOptions }} MetaItem
```

Comment:
There are cases when we do not specify `inputOptions`, therefore we should mark it as optional.

```suggestion
 * @typedef {{ label: string; value: string, inputOptions?: InputOptions }} MetaItem
```